Gems/Atom/RPI/Code/Source/RPI.Private/RPISystemComponent.cpp

```diff
@@ -84,8 +86,11 @@ namespace AZ
 
         void RPISystemComponent::Activate()
         {
-            // [GFX TODO] [ATOM-1436] this can be removed when setup and save system component's configure from projectConfigure.exe is fixed.
-            m_rpiDescriptor.m_rhiSystemDescriptor.m_drawListTags.push_back(AZ::Name("forward"));
+            auto settingsRegistry = AZ::SettingsRegistry::Get();
+            if (settingsRegistry)
+            {
+                settingsRegistry->GetObject(m_rpiDescriptor, "/O3DE/Atom/RPI/Initialization");
```

Comment:
Gem Registry in the game launcher won't be processed at this point.
So it would fail loading when building from a clean cache.
The issue is not exposed here is because default values are the same as in the data file. I suggest give a warning if GetObject fails.

The game launcher issue should be fixed by this PR:
https://github.com/o3de/o3de/pull/3690

Reply:
That's the issue with setting registry but not where it's used. 
Also, we allow people use setreg merge which may delete the object from setreg. I don't think a warning is necessary in this case.